python/cudf/cudf/core/column/categorical.py

```diff
@@ -129,7 +130,7 @@ def codes(self) -> "cudf.Series":
         return cudf.Series(self._column.codes, index=index)
 
     @property
-    def ordered(self) -> bool:
+    def ordered(self) -> Optional[bool]:
```

Comment:
Why is this `Optional`? Doesn't this always return True or False?

Reply:
`None` (at least in Pandas) has different semantics from `False` (https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.CategoricalDtype.html).

I'm not sure if we actually support it, but currently in cuDF, we do allow `ordered=None`:

```
In [15]: import cudf

In [16]: dt = cudf.CategoricalDtype(categories=[1, 2, 3])

In [17]: dt.ordered is None
Out[17]: True
```